### PR TITLE
CLDR-15366 BRS v41 currency codes

### PR DESCRIPTION
**Pull request by srl295**
- VED new, but transitional.

https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/amendments/dl_currency_iso_amendment_170.pdf

CLDR-15366

- [ ] This PR completes the ticket.

### CODE REVIEW

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/ISO4217.txt [hunk 1]
```diff
@@ -400,3 +400,4 @@ currency   	|	CNH	|	Yuan Renminbi (offshore)	|	CN	|	CHINA	|	F
 currency	|	STD	|	Dobra	|	ST	|	SÃO TOME AND PRINCIPE	|	O
 currency	|	VEF	|	Bolivar Fuerte	|	VE	|	VENEZUELA	|	O
 currency	|	UYW	|	Uruguayan Nominal Wage Index Unit	|	UY	|	URUGUAY	|	F
+currency    |   VED |   Bolívar Soberano    |  VE   |  VENEZUALA   | F
```
Comment (srl295) [inline tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/ISO4217.txt:403]: added manually

**Review by srl295 — COMMENTED** [review]
@macchiati  any idea why GenerateValidityXml did not add `VED` to currency.xml? followed all the steps.

File: common/supplemental/supplementalData.xml [hunk 2]
```diff
@@ -1112,6 +1112,7 @@ The printed version of ISO-4217:2001
             <currency iso4217="VES" from="2018-08-20"/>
             <currency iso4217="VEF" from="2008-01-01" to="2018-08-20"/>
             <currency iso4217="VEB" from="1871-05-11" to="2008-06-30"/>
+			<currency iso4217="VED" tender="false"/> <!-- transitional -->
```
Comment (srl295) [inline common/supplemental/supplementalData.xml:1115]: > for any internal needs during the redenomination process, but is not replacing VES as the official currency code 

So is this the right

**Review by pedberg-icu — DISMISSED** [review]

**Comment by pedberg-icu** [comment]
There is this failure:
```
TestDisplayAndInputProcessor {
[526](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:526)
    TestAExemplars (0.042s) Passed
[527](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:527)
    TestAdlamNasalization (0.076s) Passed
[528](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:528)
    TestAll
[529](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:529)
Error:  (TestDisplayAndInputProcessor.java:325)  Error: en	No roundtrip in DAIP:
[530](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:530)
	  value<Bolívar Soberano>
[531](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:531)
	display<Bolívar Soberano>
[532](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:532)
	  input<Bolívar Soberano>
[533](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:533)
	   diff<?>
[534](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:534)
	path<//ldml/numbers/currencies/currency[@type="VED"]/displayName[@count="one"]>
[535](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:535)

[536](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:536)
Error:  (TestDisplayAndInputProcessor.java:325)  Error: en	No roundtrip in DAIP:
[537](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:537)
	  value<Bolívar Soberano>
[538](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:538)
	display<Bolívar Soberano>
[539](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:539)
	  input<Bolívar Soberano>
[540](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:540)
	   diff<?>
[541](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:541)
	path<//ldml/numbers/currencies/currency[@type="VED"]/displayName>
[542](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:542)

[543](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:543)
Error:  (TestDisplayAndInputProcessor.java:325)  Error: en	No roundtrip in DAIP:
[544](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:544)
	  value<Bolívar Soberanos>
[545](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:545)
	display<Bolívar Soberanos>
[546](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:546)
	  input<Bolívar Soberanos>
[547](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:547)
	   diff<?>
[548](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:548)
	path<//ldml/numbers/currencies/currency[@type="VED"]/displayName[@count="other"]>
[549](https://github.com/unicode-org/cldr/runs/5296144085?check_suite_focus=true#step:14:549)
```

File: common/bcp47/currency.xml [hunk 3]
```diff
@@ -277,6 +277,7 @@ For terms of use, see http://www.unicode.org/copyright.html
             <type name="veb" description="Venezuelan Bolívar (1871-2008)" since="1.9"/>
             <type name="vef" description="Venezuelan Bolívar (2008-2018)"/>
             <type name="ves" description="Venezuelan Bolívar" since="34"/>
+            <type name="ved" description="Bolívar Soberano" since="41"/>
```
Comment (macchiati) [inline common/bcp47/currency.xml:280]: The text from ISO was not in NFC:
https://r12a.github.io/app-conversion/

"Boli&#x0301;var Soberano"

**Review by macchiati — CHANGES_REQUESTED** [review]

File: common/bcp47/currency.xml (hunk 3 above)
Comment (macchiati) [reply common/bcp47/currency.xml:280]: Whoops, wrong line:
"Boli\u{301}var Soberano"

Comment (srl295) [reply common/bcp47/currency.xml]: Thanks. Fixed, and also filed a ticket to improve the err message: CLDR-15386

**Review by pedberg-icu — APPROVED** [review]

**Review by macchiati — APPROVED** [review]

**Pull request merged**